cranelift/codegen/src/machinst/lower.rs

```diff
@@ -502,6 +508,10 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
 
         let mut value_ir_uses = SecondaryMap::with_default(ValueUseState::Unused);
 
+        if let Some(sret_param) = sret_param {
+            value_ir_uses[sret_param] = ValueUseState::Once;
```

Comment:
Could we add a comment here to note it starts at `Once` (for the one use that is the implicit return), but can still be updated below?

Reply:
Actually, should I just start it at `Multiple` in case there are multiple epilogues? The definition of the sret vreg can't participate in load-sinking or anything so I don't think the value really matters, but I guess it should be "correct" anyway?

Reply:
Oh, yes, that makes more sense actually, I agree. It's unlikely to matter as you say but might as well get the analysis right...